client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -62,22 +66,35 @@ const SecurePaymentForm = React.createClass( {
 			return 'free-trial';
 		} else if ( this.state && this.state.userSelectedPaymentBox ) {
 			return this.state.userSelectedPaymentBox;
-		} else if ( cartValues.isCreditCardPaymentsEnabled( cart ) ) {
-			return 'credit-card';
-		} else if ( cartValues.isPayPalExpressEnabled( cart ) ) {
-			return 'paypal';
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods.primary ) ) {
+			return preferredPaymentMethods.primary;
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods.secondary ) ) {
+			return preferredPaymentMethods.secondary;
 		}
 
 		return null;
 	},
 
+	getLocalizedPaymentMethodDefaults( locale, countryCode ) {
+		const default_payment_methods = { primary: 'credit-card', secondary: 'paypal' };
```

Comment:
Rather than assuming exactly 2, maybe we should have an ordered list, and then this UI element can pull the first two that are enabled?

In addition to being more future proof, it lets us describe locales in terms of each other (e.g. take the US and remove `credit-card` and/or add `bank-note`).